internal/core/adt/expr.go

```diff
@@ -1481,6 +1481,7 @@ func (x *CallExpr) evaluate(c *OpContext, state vertexStatus) Value {
 	if b.IsValidator(len(args)) {
 		return &BuiltinValidator{x, b, args}
 	}
+	c.stats.Builtins++
```

Comment:
note that BuiltinValidator ends up calling the `call` method as well; you should probably increment the counter inside that method to capture all builtin calls being performed.